tests/security/cc/ip_eb_tables.pm

```diff
@@ -23,9 +23,34 @@ sub run {
 
     select_console 'root-console';
 
+    #Configure bridge for ip_eb_tables workaround
+    assert_script_run("echo \"BOOTPROTO='dhcp'\" > /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"STARTMODE='auto'\" >> /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"BRIDGE='yes'\" >> /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"BRIDGE_PORTS='eth0'\" >> /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"BRIDGE_STP='off'\" >> /etc/sysconfig/network/ifcfg-br0");
+    assert_script_run("echo \"BRIDGE_FORWARDDELAY='15'\" >> /etc/sysconfig/network/ifcfg-br0");
+    
+    # Backup eth0 configuration
+    assert_script_run("cp /etc/sysconfig/network/ifcfg-eth0 /etc/sysconfig/network/ifcfg-eth0.bak");
+
+    # eth0 coniguration
+    assert_script_run("echo \"IPADDR='0.0.0.0'\" > /etc/sysconfig/network/ifcfg-eth0");
+    assert_script_run("echo \"BOOTPROTO='none'\" >> /etc/sysconfig/network/ifcfg-eth0");
+    assert_script_run("echo \"STARTMODE='auto'\" >> /etc/sysconfig/network/ifcfg-eth0");
```

Comment:
Could you please use a variable to invoke `/etc/sysconfig/network/ifcfg-eth0` and `/etc/sysconfig/network/ifcfg-br0`?
e.g.,
my $f_ifcfg-br0 = '/etc/sysconfig/network/ifcfg-br0';
assert_script_run("echo \"BOOTPROTO='dhcp'\" > $$f_ifcfg-br0 ");

Reply:
Fixed